iothub/device/src/Transport/Amqp/AmqpAuthenticationRefresher.cs

```diff
@@ -78,10 +86,13 @@ private async Task RefreshLoopAsync(DateTime refreshesOn, CancellationToken canc
             while (!cancellationToken.IsCancellationRequested)
             {
                 if (Logging.IsEnabled)
-                    Logging.Info(this, refreshesOn, $"Before {nameof(RefreshLoopAsync)}");
+                    Logging.Info(this, refreshesOn, $"{_amqpIotCbsTokenProvider} Before {nameof(RefreshLoopAsync)}");
```

Comment:
```suggestion
                    Logging.Info(this, refreshesOn, $"{_amqpIotCbsTokenProvider} before {nameof(RefreshLoopAsync)}");
```

Reply:
Yeah. The instance hash was useful to me when debugging multiplexed devices, so I thought it would be useful to add it into the log.